pkg/apiserver/docs/swagger.yaml

```diff
@@ -309,8 +336,8 @@ paths:
     get:
       consumes:
       - application/json
-      description: List all clusters. Use query parameters to filter results. Auth
-        is required
+      description: List all clusters. Use query parametricsers to filter results.
```

Comment:
Are swagger.`{json|yaml}` generated from `docs.go`? If so, please correct the issue there and re-generate these files too.